### PR TITLE
Bread remove error. 

### PR DESCRIPTION
**Pull request by kvlknctk**
check row emtpy. it is searching json data in row.

### CODE REVIEW

File: src/Http/Controllers/VoyagerBreadController.php [hunk 1]
```diff
@@ -393,8 +393,10 @@ protected function cleanup($dataType, $data)
 
         // Delete Files
         foreach ($dataType->deleteRows->where('type', 'file') as $row) {
-            foreach (json_decode($data->{$row->field}) as $file) {
-                $this->deleteFileIfExists($file->download_link);
+            if ($row->field == "") {
```
Comment (fletch3555) [inline src/Http/Controllers/VoyagerBreadController.php:396]: I THINK you might have the logic backwards here...

**Review by fletch3555 — CHANGES_REQUESTED** [review]
Please change the `==` to `!=` per my other comment

**Comment by fletch3555** [comment]
@kvlknctk, you have merge conflicts and an outstanding change request.  Once these are resolved, we'll review and merge it.

**Comment by handiwijoyo** [comment]
@kvlknctk Will you update this?

**Comment by fletch3555** [comment]
No response

**Pull request closed without merging**